packages/framework/collabspaces/src/test/collabSpaces.spec.ts

```diff
@@ -319,6 +329,201 @@ describe("Temporal Collab Spaces", () => {
 		assert(value2?.value === initialValue, "value was preserved correctly");
 	});
 
+	describe("Reverse Mapping tests", () => {
+		function compareMaps(
+			map1: { [id: string]: number },
+			map2: { [id: string]: number },
+		): boolean {
+			assert(Object.keys(map1).length === Object.keys(map2).length, "maps size is different");
+			for (const [id, rowValue] of Object.entries(map1)) {
+				if (id in map2) {
+					const colValue = map2[id];
+					assert(
+						rowValue === colValue,
+						`Values for id ${id} do not match: row = ${rowValue}, col = ${colValue}`,
+					);
+				} else {
+					assert(false, `No matching id found in map2 for ${id}`);
+				}
+			}
+			return true;
```

Comment:
function should return void - you never check its return value, and it's always true, so it's useless